src/sqlancer/stonedb/gen/StoneDBInsertGenerator.java

```diff
@@ -0,0 +1,88 @@
+package sqlancer.stonedb.gen;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import sqlancer.Randomly;
+import sqlancer.common.query.ExpectedErrors;
+import sqlancer.common.query.SQLQueryAdapter;
+import sqlancer.common.schema.AbstractTableColumn;
+import sqlancer.stonedb.StoneDBProvider.StoneDBGlobalState;
+import sqlancer.stonedb.StoneDBSchema.StoneDBColumn;
+import sqlancer.stonedb.StoneDBSchema.StoneDBTable;
+import sqlancer.stonedb.StoneDBToStringVisitor;
+
+public class StoneDBInsertGenerator {
```

Comment:
Would it be applicable to inhert from https://github.com/sqlancer/sqlancer/blob/1f80de16e38c4a144e8dabbcef00d07cd15470f9/src/sqlancer/common/gen/AbstractInsertGenerator.java here?

Reply:
will try